contributing.md

```diff
@@ -137,17 +137,17 @@ conda activate plotly-dev
 
 ### Install requirements - (Non-Windows)
 ```bash
-(plotly_dev) $ pip install -r packages/python/plotly/requirements.txt
-(plotly_dev) $ pip install -r packages/python/plotly/optional-requirements.txt
+(plotly_dev) $ pip install -r packages/python/plotly/requires-install.txt
```

Comment:
are these related to the pandas warning change?

Reply:
No, this is related to the updates to requirements files.